framework/include/meshgenerators/SideSetLowerDElementsGenerator.h

```diff
@@ -14,12 +14,12 @@
 /**
  * Add lower dimensional elements along the faces contained in a side set
  */
-class MeshSideSetGenerator : public MeshGenerator
+class SideSetLowerDElementsGenerator : public MeshGenerator
```

Comment:
I don't know how this happened, but we should delete this object I think. `LowerDBlockFromSidesetGenerator` does the same thing it appears and works with distributed meshes (although there are still a few tweaks that I need to make)

Reply:
ok deleted. We'll see what breaks and who's using this badly named old one